data/src/test/java/com/microsoft/azure/kusto/data/ConnectionStringBuilderTest.java

```diff
@@ -118,4 +118,22 @@ void createWithAadApplicationCertificate() throws CertificateException, Operator
                 () -> ConnectionStringBuilder
                         .createWithAadApplicationCertificate("resource.uri", "appID", x509Certificate, null));
     }
+
+    @Test
+    @DisplayName("validate createWithAadUserTokenAuthentication throws IllegalArgumentException exception when missing or invalid parameters")
+    void createWithAadUserTokenAuthentication(){
+
+        Assertions.assertThrows(IllegalArgumentException.class,
+                () -> ConnectionStringBuilder
+                        .createWithAadUserTokenAuthentication(null, "token"));
+        Assertions.assertThrows(IllegalArgumentException.class,
+                () -> ConnectionStringBuilder
+                        .createWithAadUserTokenAuthentication("","token"));
+        Assertions.assertThrows(IllegalArgumentException.class,
+                () -> ConnectionStringBuilder
+                        .createWithAadUserTokenAuthentication("resource.uri", null));
+        Assertions.assertThrows(IllegalArgumentException.class,
+                () -> ConnectionStringBuilder
```

Comment:
Can you please add a positive test, where we initiate the CSB with valid token, and we get it when we try?

Reply:
we dont do it anywhere else but sure i can

Reply:
I meant more like Assertions.Equels(ConnectionStringBuilder           .createWithAadAccessTokenAuthentication("resource.uri","token").getAccessToken(), "token")